src/escalus_stanza.erl

```diff
@@ -18,6 +18,7 @@
 
 %% old ones
 -export([id/0,
+         message/4,
```

Comment:
Why in the `old ones` section, though? Is it only temporary / intended to be removed shortly?